core/DAPS/README.md

```diff
@@ -1,16 +1,212 @@
+
+
+
+## Request call to get a token
+
+`client_assertion` id the base64 encoded request token, shown under
+ ["Request token that is handed in at DAPS side"](#request-token-that-is-handed-in-at-daps-side).
+
+
+|**FormBody Attribute**|**content**
+|:---|:---|
+|**`grant_type`**  | OAuth based grant type. See [TODO : client_credentials grant](TODO). |
+|**`client_assertion_type`**  | See [JSON Web Token (JWT) Profile for OAuth 2.0 Client Authentication and Authorization Grants](https://tools.ietf.org/html/rfc7523). |
+|**`client_assertion`**  | The signed and base64 encoded request token. Paste the example to jwt.io to see the decoded JWT. The token is signed with the connectors private key belonging to the public key contained in the X.509 certificate. |
+
+
+### Example of a "request call to get a token"
+
+```http request
+POST /
+Host: https://daps.example.com
+Content-Type: application/x-www-form-urlencoded
+
+"grant_type=client_credentials
+ &client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+ &client_assertion=eyJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJkZW1vY29ubmVjdG9yMSIsInN1YiI6ImRlbW9jb25uZWN0b3IxIiwiZXhwIjoxNTQ4Nzg1Mzg2LCJuYmYiOjE1NDg3ODE3ODYsImlhdCI6MTU0ODc4MTc4NiwiYXVkIjoiaHR0cHM6Ly9hcGkubG9jYWxob3N0In0.JSQuMf-9Fd7DNna_-s-sR7eXgcSYNCau5WgurrGJTuCSLKqhZe3odXfunN2vRFgUhU21ADFlEq96mlbQDueBlMtaXrcHFPSpIUtvuIMIVqQcGYkDdSJr_VmDuAykCYpyTCkLa7a8DTV-N3sECp-AxUgmEzYIfh8jW0WS6ehgUzrnpH6t_h_GWXKkNSAg3ERakDc4NY02pBGmiN7bmtLZNt5b4LWALiiFiduC7JbIpx4awOU6skMApmzgLnZmmTG20JlJRg6hAqyHEz5Cd4rUgrt0twmpC0Us_CG23KdUF5fWI55dcO2qAVvhNQXpqz7IiPcF7-jgkrx4oukYNY6eHA"
+ &scope=ids_connector_attributes"
+  
+```
+ 
+> **!!!** REMARK: NO line breaks in front of '&', done for better readability only **!!!**
+
+
+See also:
+- [JSON-LD](https://en.wikipedia.org/wiki/JSON-LD)
+- [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier)
+
+---
+
+
+## Dynamic Attribute Token Content
+
+The DAPS issues the requested DAT, if authentication succeeds.
+
+The DAT has these header fields:
+
+|**Field name**|**mandantory**|**cardinality**|**content**
+|:---|:---|:---|:---|
+|**`typ`**             | yes    | 1    | The token type. Must be "JWT". |
+|**`kid`**             | yes    | 1    | Key id used to sign that token. Must match the jwks.json entry found at daps-url/.well-known/jwks.json |
+|**`alg`**             | yes    | 1    | Algorithm used to sign the token.   |
+
+The DAT has these payload fields:
+
+|**Field name**|**mandantory**|**cardinality**|**content**
+|:---|:---|:---|:---|
+|**`@context`**             | yes    | 1    | The JSON-LD context containing the IDS classes, properties and instances. Must be "https://w3id.org/idsa/contexts/context.jsonld". |
+|**`@type`**                | yes    | 1    | In the context of the IDS, the request payload is an RDF instance and therefore must state that it has "@type" : "ids:DatRequestToken". TODO: @SBA: We need a type def for this... |
```

Comment:
TODO in text

Reply:
TODO means TODO. DRAFT-branch!

Reply:
Well, I should review if the DRAFT-Branch-PullRequest can in the meantime be merged into the master, whereby that would also be on the master ;)